docset/windows/netadapter/Get-NetAdapter.md

```diff
@@ -56,82 +56,81 @@ To see more information regarding the miniport, device driver, such as driver da
 ## EXAMPLES
 
 ### Example 1: Get all visible network adapters
-```
+```powershell
 PS C:\> Get-NetAdapter -Name "*"
 ```
 
 This command gets all of the visible network adapters.
 
 ### Example 2: Get all visible and hidden network adapters
-```
+```powershell
 PS C:\> Get-NetAdapter -Name "*" -IncludeHidden
 ```
 
 This command gets all of the network adapters.
 
 ### Example 3: Get all physical network adapters
-```
+```powershell
 PS C:\> Get-NetAdapter -Name "*" -Physical
 ```
 
 This command gets all of the physical network adapters.
 
 ### Example 4: Get a network adapter by the specified name
-```
+```powershell
 PS C:\> Get-NetAdapter -Name "Ethernet 2"
 
-
-This command is a version that uses wildcard characters.
 PS C:\> Get-NetAdapter -Name "E*2"
 ```
+The first command gets the network adapter named Ethernet 2.
 
-This command gets the network adapter named Ethernet 2.
```

Comment:
I separated it because the part of the description was in the code block and then it did not seems suitable to use "this command" as there were actually two commands

Reply:
Are you guys positive on splitting this example into two?

Reply:
Usually an example shows some usage of the cmdlet and there is not description text in the code block, that is why I moved it out. Probably we can split it in two examples as it actually shows two possible usages of the cmdlet...